erts/emulator/sys/common/erl_check_io.h

```diff
@@ -112,6 +112,22 @@ typedef struct {
 } ErtsIoTask;
 
 
+ERTS_GLB_INLINE int erts_sched_poll_enabled(void);
+
+#if ERTS_GLB_INLINE_INCL_FUNC_DEF
+
+ERTS_GLB_INLINE int erts_sched_poll_enabled(void)
+{
+#if ERTS_POLL_USE_SCHEDULER_POLLING
```

Comment:
Should we rename this define to `ERTS_POLL_ENABLE_SCHEDULER_POLLING` or something similar? I was confused at first as to why the define was needed at all.